packages/sdks/snippets/angular/src/app/app.module.ts

```diff
@@ -17,8 +18,10 @@ import { CatchAllComponent } from './catch-all/catch-all.component';
   imports: [
     BrowserModule,
     Content,
+    BlogArticleComponent,
     RouterModule.forRoot([
       { path: 'announcements/:id', component: AnnouncementBarComponent },
+      { path: 'blogs/:handle', component: BlogArticleComponent },
```

Comment:
do we need dynamic `:handle` if we are hardcoding and using query as `'new-product-line'`?